<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4P44GED" ignored="0" name="Test1" tests="4" failures="1" timestamp="2020-11-18T09:49:32 CST" time="63.756" errors="0">
  <testcase name="testLogoOff" time="15.595" classname="com.test.demo.SmokeTesting"/>
  <testcase name="testLogoOn" time="15.456" classname="com.test.demo.SmokeTesting"/>
  <testcase name="testContainerOff" time="0.01" classname="com.test.demo.SmokeTesting"/>
  <testcase name="testContainerOn" time="15.342" classname="com.test.demo.SmokeTesting">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.demo.SmokeTesting.testContainerOn(Unknown Source)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testContainerOn -->
</testsuite> <!-- Test1 -->
